src/composables/Alchemy.ts

```diff
@@ -1,6 +1,10 @@
+import { isNumber } from "windicss/utils";
+import { Growth } from "./Utilities";
+
 export type AlchemyData = {
   vials: Record<string, number>;
   upgrades: Record<Color, number[]>;
+  goals: Record<Color, number>[];
```

Comment:
```suggestion
  goals: Record<Color, number[]>;
```
It should be Record from Color to number[] not an array of Record from Color to number

Reply:
This has been applied